static/app/components/platformPicker.tsx

```diff
@@ -182,27 +179,13 @@ const NavContainer = styled('div')`
   border-bottom: 1px solid ${p => p.theme.border};
 `;
 
-const SearchBar = styled('div')`
-  ${p => inputStyles(p)};
-  padding: 0 8px;
-  color: ${p => p.theme.subText};
-  display: flex;
-  align-items: center;
-  font-size: 15px;
-  margin-top: -${space(0.75)};
-
-  input {
-    border: none;
-    background: none;
-    padding: 2px 4px;
-    width: 100%;
-    /* Ensure a consistent line height to keep the input the desired height */
-    line-height: 24px;
-
-    &:focus {
-      outline: none;
-    }
-  }
+const StyledSearchBar = styled(SearchBar)`
+  min-width: 6rem;
```

Comment:
Little thing I noticed - since this doesn't have a set width, the input gets larger when you type into it (because of the added clear button). Seemed a little jarring

Reply:
Ahh good catch! This seems to be a Chrome/Webkit-specific issue. I was developing in Firefox and didn't see it. Fixed in https://github.com/getsentry/sentry/pull/38067/commits/fe3e33cf499aeeea5cd450d7aaba8e706bc75c50.